.github/workflows/pr_checks.yml

```diff
@@ -0,0 +1,156 @@
+        with:
+          name: ${{ env.CONTAINER }}
+          path: /tmp/
+
+      - name: Load fidesctl image
+        run: docker load --input /tmp/${{ env.CONTAINER }}.tar
+
+      - name: Check fidesctl installation
+        run: docker run ${{ env.IMAGE }} fidesctl
+
+      - name: Run fidesctl evaluation
+        run: docker run --rm ${{ env.IMAGE }} fidesctl --local evaluate fides_resources/
+
+  Format:
+    needs: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download fidesctl container
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ env.CONTAINER }}
+          path: /tmp/
+
+      - name: Load fidesctl image
+        run: docker load --input /tmp/${{ env.CONTAINER }}.tar
+
+      - name: Run formatter
+        run: docker run ${{ env.IMAGE }} black --check src/
+
+  Lint:
+    needs: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download fidesctl container
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ env.CONTAINER }}
+          path: /tmp/
+
+      - name: Load fidesctl image
+        run: docker load --input /tmp/${{ env.CONTAINER }}.tar
+
+      - name: Run linter
+        run: docker run ${{ env.IMAGE }} pylint src/
+
+  TypeCheck:
+    needs: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download fidesctl container
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ env.CONTAINER }}
+          path: /tmp/
+
+      - name: Load fidesctl image
+        run: docker load --input /tmp/${{ env.CONTAINER }}.tar
+
+      - name: Run typechecker
+        run: docker run ${{ env.IMAGE }} mypy
+
+  Complexity:
+    needs: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download fidesctl container
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ env.CONTAINER }}
+          path: /tmp/
+
+      - name: Load fidesctl image
+        run: docker load --input /tmp/${{ env.CONTAINER }}.tar
+
+      - name: Run cyclomatic complexity check
+        run: docker run ${{ env.IMAGE }} xenon src --max-absolute B --max-modules B --max-average A --ignore "data, tests, docs" --exclude "src/fidesctl/core/annotate_dataset.py,src/fidesctl/_version.py"
+
+  Test:
+    needs: Build
+    runs-on: ubuntu-latest
+    services:
+      fidesctl-db:
+        image: postgres:12
+        env:
+          POSTGRES_DB: fidesctl
+          POSTGRES_PASSWORD: fidesctl
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 5s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+      - name: Download fidesctl container
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ env.CONTAINER }}
+          path: /tmp/
+
+      - name: Load fidesctl image
+        run: docker load --input /tmp/${{ env.CONTAINER }}.tar
+
+      - name: Start fidesctl
+        run: docker run ${{ env.IMAGE }}
```

Comment:
This step consistently fails because the fidesctl webserver can't find the Postgres database. I think I might be missing some config/environment variables in the `services:` section? Additionally, I think there's a logical flaw in this step, because in a properly configured version, it would simply call `fidesctl webserver` which is a long-lived command. Asking the job to next start the container _again_ + run the `pytest` check likely wouldn't work. I'm not sure how to work through either issue.